### PR TITLE
Added flowchart for Docs overview

### PR DESCRIPTION
**Pull request by kolibril13**
This flowchart, made by @leotrs provides an overview of what and where to find things in the docs.

In the flowchart there was the option "embed as HTML".
I will try this first, but if that does not work, we can include this as an png.
EDIT: 
Link to the flowchart: ``https://app.diagrams.net/#G1aKyJTloYB97IhrzwqEENOu-WQyuVWMjM``

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -223,5 +223,20 @@ Further useful guidelines
 #. Please do as much as possible to keep issues, PRs, and development in
    general as tidy as possible.
 
+URL = `https://app.diagrams.net/#G1aKyJTloYB97IhrzwqEENOu-WQyuVWMjM`
+
+
+Docs workflow chart:
+<div class="mxgraph" style="max-width:100%;border:1px solid transparent;" data-mxgraph="{&quot;highlight&quot;:&quot;#0000ff&quot;,&quot;nav&quot;:true,&quot;resize&quot;:true,&quot;toolbar&quot;:&quot;zoom layers lightbox&quot;,&quot;edit&quot;:&quot;_blank&quot;,&quot;url&quot;:&quot;https://drive.google.com/uc?id=1aKyJTloYB97IhrzwqEENOu-WQyuVWMjM&amp;export=download&quot;}"></div>
```
Comment (naveen521kk) [inline docs/source/contributing.rst:230]: Please don't use your drive url. Instead add the png file to source tree.

**Review by naveen521kk — CHANGES_REQUESTED** [review]

**Comment by kolibril13** [comment]
To clarify why I chose this approach:
* As pngs have mostly a size of ~Mb , this HTML approach produces less memory in the repo (Especially when pngs are altered, this quickly goes up) 
* This is easier to maintain: If we want to change something in this flowchart, we can simply go to the flowchart and edit it on app.diagrams.net (probably we should find a way that this link is only accessible for community devs.)
* On the website, this just looks *delightful*: There is a nice increase button, you can zoom, and shift around etc.

**Comment by leotrs** [comment]
Looks great! I just tried editing the flowchart via the URL you linked and it shows immediately :+1:

**Review by leotrs — APPROVED** [review]

**Review by naveen521kk — APPROVED** [review]
LGTM! Also it works.

**Comment by kolibril13** [comment]
Links to the diagrams:
https://app.diagrams.net/#G1aKyJTloYB97IhrzwqEENOu-WQyuVWMjM
https://app.diagrams.net/#G1WYVKKoRbXrumHEcyQKQ9s1yCnBvfU2Ui

**Pull request merged**